source/extensions/tracers/opentelemetry/config.cc

```diff
@@ -0,0 +1,44 @@
+#include "source/extensions/tracers/opentelemetry/config.h"
+
+#include "envoy/config/trace/v3/opentelemetry.pb.h"
+#include "envoy/config/trace/v3/opentelemetry.pb.validate.h"
+#include "envoy/registry/registry.h"
+
+#include "source/common/common/logger.h"
+#include "source/extensions/tracers/opentelemetry/opentelemetry_tracer_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+OpenTelemetryTracerFactory::OpenTelemetryTracerFactory()
+    : FactoryBase("envoy.tracers.opentelemetry") {}
+
+Tracing::DriverSharedPtr OpenTelemetryTracerFactory::createTracerDriverTyped(
+    const envoy::config::trace::v3::OpenTelemetryConfig& proto_config,
+    Server::Configuration::TracerFactoryContext& context) {
+  // Since OpenTelemetry can only support a single tracing configuration per entire process,
+  // we need to make sure that it is configured at most once.
```

Comment:
Where is this limitation comes from? Also this need to be documented in proto file.

Reply:
Ah on further review, I think I was fooling myself by following the pattern in the OpenCensus tracer. I don't believe this limitation applies here, and I've removed it. Thanks!